<h3>Session: Challenges</h3>

<div class="schedulerow">
    <div class="timeSpans">14:00 - 14:05</div>
    <div class="scheduledetails">
        <div class="well"><strong>Opening Presentation</strong></div>
        <div class="room"><strong>Speaker:</strong> Daniel Jönsson</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">14:05 - 14:25</div>
    <div class="scheduledetails">
        <div class="well"><strong>Challenge Introduction: Deciphering Protein Modifications with the Power of
                Visualization</strong></div>
        <div class="room"><strong>Speaker:</strong> Enrico Massignani, Flanders Institute for Biotechnology</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
        <details>
            <summary>Abstract and Bio</summary>
            <p><strong>Abstract:</strong> Post-Translational Modification (PTM) is the covalent addition or removal of
                functional groups from an amino acid residue in a protein. Thus, PTMs contribute greatly to the
                complexity of the proteome by generating millions of potential proteoforms from a few thousands of
                unmodified proteins. PTMs play a key regulatory role in every known biological process - from signal
                transduction to mRNA splicing - and their dysregulation is often a hallmark of disease. An important
                aspect is that multiple PTMs can co-exist on the same protein, histones being a prime example of this
                crosstalk. Within this context, one of the projects of the CompOmics group is reprocessing proteomics
                data from the PRIDE repository using “ionbot”, a machine learning-based peptide identification engine
                that can perform “open PTM searches” to identify all PTMs that are present in a sample. Although the
                project has successfully found new PTMs, the challenge we face now is finding a clear and concise
                visualization to communicate and interpret our results.</p>
            <p><strong>Bio:</strong> Enrico Massignani studied Biotechnology and Bioinformatics at the University of
                Milan (Italy). He learned my PhD in 2021 with a thesis on computational methods to research protein
                arginine methylations and other PTMs by mass spectrometry, supervised by Dr Tiziana Bonaldi at the
                European School of Medicine (SEMM, Milan, Italy). He is currently working as a Postdoc at the Flanders
                Institute for Biotechnology, in the CompOmics group led by Prof Lennart Martens. His research mainly
                focuses on understanding the effects of PTMs on a protein's structure and function and how different
                PTMs can interact with each other.</p>
        </details>
    </div>
</div>

<h3>Challenge talks</h3>

<div class="schedulerow">
    <div class="timeSpans">14:25 - 14:30</div>
    <div class="scheduledetails">
        <div class="well"><strong>EProM - Exploration of Protein Modifications</strong></div>
        <div class="room"><strong>Speaker:</strong> Daniël M. Bot</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">14:30 - 14:35</div>
    <div class="scheduledetails">
        <div class="well"><strong>Mouse-Human Hybrids</strong></div>
        <div class="room"><strong>Speaker:</strong> Hauke Bartsch</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">14:35 - 14:40</div>
    <div class="scheduledetails">
        <div class="well"><strong>ANNOTED: Taming Protein Beasts Through Interactive Visualization</strong></div>
        <div class="room"><strong>Speaker:</strong> Aaron Watters</div>
        <div class="room"><strong>Type:</strong> Virtual Presentation (pre-recorded)</div>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">14:40 - 14:45</div>
    <div class="scheduledetails">
        <div class="well"><strong>A simple pipeline for creating cover quality images of post-translational
                modifications to molecular structures in rare disease</strong></div>
        <div class="room"><strong>Speaker:</strong> Mo Rahman</div>
        <div class="room"><strong>Type:</strong> Virtual Presentation (pre-recorded)</div>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">14:45 - 14:50</div>
    <div class="scheduledetails">
        <div class="well"><strong>ProtoFold Neighborhood Inspector</strong></div>
        <div class="room"><strong>Speaker:</strong> Nicolas F. Chaves-de-Plaza</div>
        <div class="room"><strong>Type:</strong> Virtual Presentation (pre-recorded)</div>
    </div>
</div>

<h3>Redesign talks</h3>

<div class="schedulerow">
    <div class="timeSpans">14:50 - 14:55</div>
    <div class="scheduledetails">
        <div class="well"><strong>Re-design challenge introduction</strong></div>
        <div class="room"><strong>Speaker:</strong> Enrico Massignani</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">14:55 - 15:00</div>
    <div class="scheduledetails">
        <div class="well"><strong>Modie Viewer: Protein Beasts and How to View Them</strong></div>
        <div class="room"><strong>Speaker:</strong> Huyen N. Nguyen</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">15:00 - 15:05</div>
    <div class="scheduledetails">
        <div class="well"><strong>(Re-design) EProM - Exploration of Protein Modifications</strong></div>
        <div class="room"><strong>Speaker:</strong> Daniël M. Bot</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">15:05 - 15:10</div>
    <div class="scheduledetails">
        <div class="well"><strong>Visualizing Protein Residue Chemical Modifications</strong></div>
        <div class="room"><strong>Speaker:</strong> Laura Garrison</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">15:10 - 15:15</div>
    <div class="scheduledetails">
        <div class="well"><strong>A Dashboard for the Visualisation of a Protein’s PTMs</strong></div>
        <div class="room"><strong>Speaker:</strong> Simon Hackl</div>
        <div class="room"><strong>Type:</strong> Virtual Presentation (pre-recorded)</div>
    </div>
</div>

<h3>Break</h3>

<div class="schedulerow">
    <div class="timeSpans">15:15 - 15:45</div>
    <div class="scheduledetails">
        <div class="well"><strong>Break</strong></div>
    </div>
</div>

<h3>Session: Invited talks</h3>

<div class="schedulerow">
    <div class="timeSpans">15:45 - 16:05</div>
    <div class="scheduledetails">
        <div class="well"><strong>Visual Analysis of Large-scale Biological Data to Understand Microbial
                Threats</strong></div>
        <div class="room"><strong>Speaker:</strong> Haichao Miao, Lawrence Livermore National Laboratory</div>
        <div class="room"><strong>Type:</strong> Virtual Presentation (live)</div>
        <details>
            <summary>Abstract and Bio</summary>
            <p><strong>Abstract:</strong> Infectious diseases pose a threat to both public health and economies
                worldwide. Dedicated cell imaging research and the development of novel drugs will be critical to
                countering these microbial threats, such as antibiotic-resistant bacteria. However, with the increasing
                capabilities of imaging techniques and the design of novel biomolecular therapeutics, the corresponding
                data are becoming increasingly difficult to analyze, and both existing data management and visualization
                are reaching their limits. In this talk, I will introduce two areas where the large scale of data and
                its spatial complexity make sense-making difficult, and discuss the challenges and possible solutions.
                In DNA nanotechnology, DNA can be used as a building material for constructing nanoscale therapeutic
                devices. However, the spatial conformations and number of features require abstract visual
                representations to effectively display information relevant to a particular in silico design task. Also,
                novel microscopes for dynamic imaging of cells, such as the lattice light sheet microscope, achieve
                unprecedented temporal resolutions and acquisition durations that can produce TB-scale data with
                incredibly high spatial complexity. Addressing these challenges goes beyond finding suitable visual
                representations but also requires appropriate solutions for data access and real-time exploration.</p>
            <p><strong>Bio:</strong> Haichao Miao is a researcher at the Center for Applied Scientific Computing,
                Lawrence Livermore National Laboratory. He focuses on applying visualization and virtual reality methods
                to problems in dynamic microscopy and additive manufacturing. He received his PhD in computer science
                from TU Wien, Austria, in 2019. From 2016 to 2020, he worked at the Austrian Institute of Technology,
                where he focused on the development of in-silico design methods for DNA nanostructure modeling.</p>
        </details>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">16:05 - 16:25</div>
    <div class="scheduledetails">
        <div class="well"><strong>A Heatmap for Enzyme Engineering: Analysis of Residue-by-Residue Contributions to the
                Free Energy Barrier of Enzyme Reactions</strong></div>
        <div class="room"><strong>Speaker:</strong> Yihan Shao, The University of Oklahoma</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
        <details>
            <summary>Abstract and Bio</summary>
            <p><strong>Abstract:</strong> In enzyme mechanistic studies and mutant designs, it is highly desirable to
                know the individual residue contributions to the reaction free energy barrier. In this talk, we will
                show that such free energy contributions from each residue can be readily obtained by post-processing ab
                initio quantum mechanical molecular mechanical (ai-QM/MM) free energy simulation trajectories.
                Specifically, through mean force integration along the minimum free energy pathway (MFEP), one can
                obtain the electrostatic, polarization, and van der Waals (vdW) contributions from each residue to the
                free energy barrier. Separately, a similar analysis procedure will allow us to assess the contribution
                from different reaction coordinates. The chorimate mutase reaction will be used to demonstrate the
                utilization of these two trajectory analysis tools. </p>
            <p><strong>Bio:</strong> Dr. Yihan Shao is an Associate Professor of Chemistry and Biochemistry at the
                University of Oklahoma. He develops computational methods and software for modeling enzyme reactions,
                such as CRISPR Cas protein catalysis, and protein covalent inhibition. He also facilitates the
                development of bioimaging (fluorescence, chemiluminescence, bioluminescence, positron emission
                tomography, and optoacoustics) probes. </p>
        </details>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">16:25 - 16:55</div>
    <div class="scheduledetails">
        <div class="well"><strong>Capstone: Bio+MedVis: Current and future opportunities</strong></div>
        <div class="room"><strong>Speaker:</strong> Barbara Kozlikova, Masaryk University</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
        <details>
            <summary>Abstract and Bio</summary>
            <p><strong>Bio:</strong> Barbora Kozlíková is an associate professor at the department of Visual Computing
                at Masaryk University in Brno, Czech Republic. She is the head of Visitlab, a visualization laboratory
                focusing on interdisciplinary research in visualization and visual analysis. She is supervising 7 PhD
                students. She received her PhD in 2011 and since then, she is working on many research projects and
                scientific visualization topics, mostly connected with biochemistry and bioinformatics. She teaches
                visualization and computer graphics at the Masaryk University.</p>
        </details>
    </div>
</div>
<div class="schedulerow">
    <div class="timeSpans">16:55 - 17:00</div>
    <div class="scheduledetails">
        <div class="well"><strong>Closing</strong></div>
        <div class="room"><strong>Speaker:</strong> Daniel Jönsson</div>
        <div class="room"><strong>Type:</strong> In Person Presentation</div>
    </div>
</div>
