---
layout: page
title: Program BioVis@ISMB
permalink: /program_ismb/
back_title: ISMB
back_url: ismb
---

# BioVis@ISMB 2022 Program

{{ site.ismb_cosi_date }}

## Invited Speakers

### Once upon a time in Bio-Medical Data Visualization: Reflections on Research Before and During Pandemic

[Tatiana von Landesberger](https://visva.cs.uni-koeln.de/landesberger),
University of Cologne

**Abstract:** Research in visualization is often motivated by the endeavor to
improve on the illustration of data: in order to better communicate data to
others and to gain deeper insights into complex datasets, possibly from a
variety of data sources. In the medical domain, the data can include for
example patient data, health records as well as biologic data such as genome.
Insights to be obtained from data may relate inter alia to the spreading of
diseases, evolutionary analysis and virus mutations. The tasks include both
retrospective analysis for finding the patient zero and modelling the spreading
of a disease, as well predictive modelling of virus mutations and future disease
spreading. The COVID-19 pandemic has confronted this general motivation for our
research to a need for practical solutions. Infection control experts needed to
quickly gain insights into novel datasets and to communicate the insights to
colleagues and to a broader public, requiring quick and efficient visualization
solutions.

New methods, tools, and methodologies have popped up from basic and from applied
research. New data was collected, model results were produced that required
rapid analysis. Multi-disciplinary teams worked and applied solutions to the new
challenges resulting from the pandemic. The rapid response was only possible by
leveraging on the experience and past research. Thus, the talk will take a
larger historical perspective and present specific solutions from own
experience, including reflections on data, task and user triangle as well as the
challenges of multidisciplinary working styles.

**Bio:** Tatiana von Landesberger is a full professor of Computer Science –
Visualization at University of Cologne, Germany. She received a PhD in 2010 and
finished her habilitation in 2017 at TU Darmstadt, Germany. Her research focuses
on information visualization and visual analytics of spatio-temporal and network
data from various domains such as biology, medicine, finance, transportation,
journalism or meteorology. Her recent research addresses the challenges of
perception and cognition of visualization as well as visual analysis of disease
spreading. She regularly publishes at top international conferences and has
received multiple awards. Tatiana has served in program committees and
organization committees for IEEE VIS, EuroVis, VMV and other venues. Recently,
she was full paper chair at EuroVis and is now member of EuroVis steering
committee and associate editor of Computer Graphics Forum.

### Machine learning provides a new perspective on protein modification

<figure class="speaker-photo">
    <img src="{{ site.baseurl }}/images/speakers/lennart_martens.jpg" alt="Lennart Martens" />
    <figcaption>Prof. Lennart Martens</figcaption>
</figure>

[Lennart Martens](https://ai.ugent.be/people/LennartMartens.en.html), Ghent
University

**Abstract:** Over the last two decades, mass spectrometry based proteomics has
evolved quite dramatically, leveraging advances in instrumentation, methods, and
software tools to dig ever deeper into the protein content of cells. Yet, as the
key effectors of cellular function, there is more to proteins than just their
sequence. Indeed, many proteins also are tightly regulated in terms of their
activity, their localisation, and their interactions. And while this is a
well-known fact, very little is actually understood about the specific
mechanisms by which this regulation is carried out. While some mechanisms such
as protein phosphorylation have been studied in quite some detail already, other
protein modifications have received much less attention, and therefore remain
largely unexplored. As a result, interesting downstream phenomena, such as
competition between, or coordination of, modifications on the same site or the
same protein remain underexplored.

Interestingly, a wholly new generation of machine learning-based identification
tools is now radically changing our ability to discover the proteome-wide
modification landscape, with [ionbot](https://ionbot.cloud) the leading exponent
of these revolutionary new tools. Itself based on the
[MS²PIP](https://iomics.ugent.be/ms2pip) and
[DeepLC](http://compomics.github.io/projects/DeepLC) models to predict analyte
behaviour, ionbot is fast and reliable, and allows unbiased identification of
protein modifications at unprecedented scale. We are now working on providing
this open modification capability to (immuno-)peptidomics as well, where we have
already been able to show that our predictors allow substantial advances in
identification sensitivity and specificity.

To show the capabilities of ionbot, and to cast a first glance at the true
complexity of a proteome as our experiments and instruments have seen it, we
have applied ionbot to over 1 billion human spectra, and over 600 million mouse
spectra from the [PRIDE archive](https://www.ebi.ac.uk/pride), uncovering a
plethora of modifications of various origine and level of interest. This wholly
new view of the modified proteome dramatically changes our view on proteins, and
shows the overwhelming abundance of modifications, chemical, biological, or
artefactual that affect the protein machinery of life.

In the past, we already leveraged such proteome-wide data to infer [protein
associations](https://iomics.ugent.be/tabloidproteome), but recent advances in
machine learning have allowed us to dramatically expand our coverage, in turn
yielding a massive increase in the (less) biased detection of protein
association or interactions from high-throughput omics data.

We can also use these proteome-wide results to analyse tissues and even cell
types for their protein composition, yielding detailed information on
tissue-specific proteomes with potentially far-reaching applications.

**Bio:** Lennart Martens is Full Professor of Systems Biology at Ghent
University, Group Leader of the Computational Omics and Systems Biology
(CompOmics) group at VIB, and Associate Director of the VIB-UGent Center for
Medical Biotechnology, all in Ghent, Belgium. He has been working in proteomics
bioinformatics since his Master’s degree, which focused on the computational
interpretation of peptide mass spectra, and the sequence-dependent
fragmentation of peptides. He then worked as a software developer and framework
architect for a software company for a few years, before returning to Ghent
University to pursue a Ph.D. in proteomics and proteomics informatics. During
this time, he worked on the development of high-throughput peptide centric
proteomics techniques and on bioinformatics tools to support these new
approaches. In 2003 he designed and built the PRIDE repository for the global
dissemination of proteomics data at EMBL-EBI as a Marie Curie fellow of the
European Commission. After obtaining his Ph.D., he rejoined EMBL-EBI to
coordinate the newly created PRIDE group for the next three years, firmly
establishing the system as the world’s foremost public proteomics data
repository. He then moved back to Ghent University and VIB to take up his
current positions, in which he focuses on novel machine learning algorithms for
mass spectrometry data analysis, and their application to the large-scale
orthogonal reprocessing of public proteomics data. Prof. Martens has been
elected to the Young Academy of the Royal Belgian Academy of Sciences in 2013,
to the Human Proteome Organisation (HUPO) Council in 2016, has been elected
President of the European Proteomics Association (EuPA) in 2020, and was
admitted as Fellow of the Royal Society for Chemistry in 2018. He also served
on the HUPO Executive Board from 2017 to 2019, and was President of the ABRF
Proteomics Informatics Research Group (iPRG) in 2011. Dr. Martens received the
2014 Prometheus Award for Research Excellence from Ghent University, and the
2015 ‘Juan Pablo Albar’ Proteomics Pioneer Award from the European Proteomics
Association (EuPA). An author on 265 peer-reviewed papers, he has also
co-written two Wiley textbooks on computational mass spectrometry.

## Program

To be announced...
